xradar/version.py

```diff
@@ -0,0 +1,4 @@
+# file generated by setuptools_scm
```

Comment:
You would need to remove version.py from the commits, we do not track it in git.